django/db/backends/base/operations.py

```diff
@@ -799,7 +799,9 @@ def explain_query_prefix(self, format=None, **options):
     def insert_statement(self, on_conflict=None):
         return "INSERT INTO"
 
-    def on_conflict_suffix_sql(self, fields, on_conflict, update_fields, unique_fields):
+    def on_conflict_suffix_sql(
+        self, fields, on_conflict, update_fields, unique_fields, where_clause
```

Comment:
Naming is hard... I'm wondering whether we should rename `where_clause` to `condition`. 🤔